tests/lightning_base/test_expval.py

```diff
@@ -188,14 +189,18 @@ def test_hermitian_expectation(self, theta, phi, tol, dev):
         ],
     )
     @pytest.mark.parametrize("dtype", [np.complex64, np.complex128])
-    def test_hermitian_arbitrary_size_expectation(self, dtype, theta, phi, n_op_wires, wires, tol):
+    def test_hermitian_arbitrary_size_expectation(
+        self, dtype, theta, phi, n_op_wires, wires, tol, seed
+    ):
         """Tests the expectation value of an arbitrary size Hermitian matrix."""
         if device_name == "lightning.tensor" and n_op_wires > 1:
             pytest.skip(
                 "lightning.tensor does not support Hermitian matrices with n_op_wires not equal to 1"
             )
         n_qubits = 8
         dev = LightningDevice(wires=n_qubits, c_dtype=dtype)
+
+        np.random.seed(seed)
```

Comment:
could you also merge `seed` with the seed variable so we have only one place to update moving forward?